contributing.md

```diff
@@ -0,0 +1,257 @@
+
+Both issue lists are sorted by total number of comments. While not perfect, number of comments is a reasonable proxy for impact a given change will have.
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain QR Scanner's quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible QR Scanner's
+- Enable a sustainable system for QR Scanner's maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Follow the [styleguides](#styleguides)
+2. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+
+## Styleguides
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+* Limit the first line to 72 characters or less
+* Reference issues and pull requests liberally after the first line
+* When only changing documentation, include `[ci skip]` in the commit title
+* Respect the commitlint rules
+
+### JavaScript Styleguide
+
+All JavaScript must adhere to [JavaScript Standard Style](https://standardjs.com/).
+
+* Prefer the object spread operator (`{...anotherObj}`) to `Object.assign()`
+* Inline `export`s with expressions whenever possible
+  ```js
+  // Use this:
+  export default class ClassName {
+
+  }
+
+  // Instead of:
+  class ClassName {
+
+  }
+  export default ClassName
+  ```
+* Place requires in the following order:
+    * Built in Node Modules (such as `path`)
+    * Local Modules (using relative paths)
+* Place class properties in the following order:
+    * Class methods and properties (methods starting with `static`)
+    * Instance methods and properties
+
+### Specs Styleguide
+
+- Include thoughtfully-worded, well-structured [Jasmine](https://jasmine.github.io/) specs in the `./tests` folder.
+- Treat `describe` as a noun or situation.
+- Treat `it` as a statement about state or how an operation changes state.
+
+#### Example
+
+```coffee
+describe 'a dog', ->
+ it 'barks', ->
+ # spec here
+ describe 'when the dog is happy', ->
+  it 'wags its tail', ->
+  # spec here
+```
+
+### Documentation Styleguide
```

Comment:
This part seems a little bit strange. In example it uses something like jsdocs (we do not use it at the moment).